native/src/window/action.rs

```diff
@@ -5,13 +5,12 @@ use std::fmt;
 
 /// An operation to be performed on some window.
 pub enum Action<T> {
-    /// Resize the window.
-    Resize {
-        /// The new logical width of the window
-        width: u32,
-        /// The new logical height of the window
-        height: u32,
-    },
+    /// Starts a window drag while mouse button is held.
+    Drag,
```

Comment:
I don't suppose there's any way we can guarantee this is called after a left button press is received? Maybe add a disclaimer similar to the one on the winit docs:

```
There’s no guarantee that this will work unless the left mouse
button was pressed immediately before this function is called.
```